### DISCUSSION TITLE
Discord is always asking to reauthorize the application

### DISCUSSION THREAD

**Discussion by davidroulier** [nextauthjs/next-auth#3490] [Help] [answered]
Hello,

Before opening a potential issue, I'd like to ask a question.

I set up Discord with Next-Auth V4 but I encounter a problem I didn't have with V3.

When clicking on "signin with discord", I'm always asked to reauthorize the application to access my information.

Is this intended or a configuration issue on my side ?

Thank you very much,
David

**Comment by balazsorban44** [comment, marked as answer]
https://discord.com/developers/docs/topics/oauth2#authorization-code-grant-authorization-url-example

**Reply by davidroulier** [reply]
> https://discord.com/developers/docs/topics/oauth2#authorization-code-grant-authorization-url-example

Thank you so much.

Considering I hadn't the issue in V3, I badly considered there might be an issue.

Working perfectly now👌

**Reply by NicholasEwing** [reply]
@davidroulier Can you explain exactly what you did? I'm trying to pass `prompt=none` but I'm new to Next-Auth and unsure of the best way to do this. I'm having the same issue you had.

**Reply by NicholasEwing** [reply]
I figured it out, for anyone wondering the `signIn()` function takes a 3rd argument that goes to [/authorize endpoint](https://next-auth.js.org/getting-started/client#additional-parameters).

```js
signIn("discord", null, { prompt: "none" });
```

**Comment by JeffinJ** [comment]
Here is the solution : https://stackoverflow.com/a/73585474/11152084
